ern-cauldron-api/src/gitstore.js

```diff
@@ -10,17 +10,33 @@ import path from 'path'
 
 const CAULDRON_FILENAME = 'cauldron.json'
 
+export type TypeCauldronCodePushMetadata = {
+  deploymentName?: string,
+  isMandatory?: boolean,
+  appVersion?: string,
+  size?: number,
+  releaseMethod?: string,
+  label?: string,
+  releasedBy?: string,
+  rollout?: number
+}
+
+export type TypeCauldronCodePushEntry = {
```

Comment:
minor : similar to above - please reevaluate the name to `CodePushEntry` only if you agree with above comment :)